mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/rg-template.json

```diff
@@ -0,0 +1,346 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Prefix used for naming resources."
+            },
+            "defaultValue": "mlos"
+        },
+        "vmSKU": {
+            "type": "string",
+            "defaultValue": "Standard_D2s_v3",
+            "metadata": {
+                "description": "VM SKU"
+            }
+        },
+        "vmAdminUsername": {
+            "type": "string"
+        },
+        "sshPublicKeys": {
+            "type": "array"
+        },
+        "storageAccountLocations": {
+            "type": "array",
+            "defaultValue": [
+                "[resourceGroup().location]"
+            ]
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "mlos-file-share"
+        },
+        "vmLocation": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "vmSshPort": {
+            "type": "string",
+            "defaultValue": "22"
+        },
+        "vmSshSourceAddressPrefix": {
+            "type": "string"
+        },
+        "storageSku": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Premium_LRS",
+                "Premium_ZRS",
+                "Standard_GRS",
+                "Standard_GZRS",
+                "Standard_LRS",
+                "Standard_RAGRS",
+                "Standard_RAGZRS",
+                "Standard_ZRS"
+            ]
+        }
+    },
+    "functions": [],
+    "variables": {
+        "storageAccountName": "[toLower(concat(replace(parameters('projectPrefix'), '-', '')))]",
```

Comment:
Ah good catch! Actually I meant to totally remove the concat because it used to add a `storage` suffix, but that was going over the character limit after adding location suffix so I removed the suffix but forgot to remove the concat. Will remove!